addons/portal/static/src/xml/portal_security.xml

```diff
@@ -0,0 +1,46 @@
+<templates xml:space="preserve">
+    <t t-name="portal.api_key_success">
+        <div>
+            <p>
+                Here is your new API key, use it instead of a
+                password when doing RPC access. Your login is still
+                necessary.
+            </p>
```

Comment:
I have some concern about the capability (and desire) of users to understand terms like API and RPC. For our own purposes, we have begun discussing some way to bootstrap this process in our iOS app (captive browser? main browser launches link into our app?) such that the user enters their password (and 2FA) to confirm this action, then the API key is stored in the keychain on the device (requiring FaceId/TouchId?) and is never really seen by the user.

Reply:
> I have some concern about the capability (and desire) of users to understand terms like API and RPC. 

The entire point of that wizard is to generate keys *for RPC access*. 

> For our own purposes, we have begun discussing some way to bootstrap this process in our iOS app (captive browser? main browser launches link into our app?) such that the user enters their password (and 2FA) to confirm this action, then the API key is stored in the keychain on the device (requiring FaceId/TouchId?) and is never really seen by the user.

Yes, you can reuse the idea and storage without using the same UI. I'm pretty sure that's what @Florimond does as their concern is oauth access from / for third-party system, so the user never sees the key. They probably edited some bits in order to make this non-interactive flow more convenient.